pkg/collector/python/datadog_agent.go

```diff
@@ -248,6 +248,10 @@ type sqlConfig struct {
 	DollarQuotedFunc bool `json:"dollar_quoted_func"`
 	// ReturnJSONMetadata specifies whether the stub will return metadata as JSON.
 	ReturnJSONMetadata bool `json:"return_json_metadata"`
+	// ObfuscateOnly specifies whether the obfuscator should only obfuscate SQL queries but not normalize them.
+	ObfuscateOnly bool `json:"obfuscate_only"`
+	// ObfuscateAndNormalize specifies whether the obfuscator should obfuscate and normalize SQL queries.
+	ObfuscateAndNormalize bool `json:"obfuscate_and_normalize"`
```

Comment:
This almost feels like this should be a single configuration (an obfuscation mode?) instead of two boolean flags that, if both set to true, would make for a conflicting configuration.

Reply:
I agree these flags are confusing, especially since if both are false, we still obfuscate right now, just with a different method.

Reply:
`obfuscation mode` is a good idea